arduino/smartcar/smartcar.ino

```diff
@@ -1,9 +1,45 @@
 #include <Smartcar.h>
 
-void setup() {
-  // put your setup code here, to run once:
+//Variable declaration
+const unsigned long PULSES_PER_METER = 40;
+const unsigned short TRIGGER_PIN = 6;
+const unsigned short ECHO_PIN = 7;
+const unsigned int MAX_DISTANCE = 500;
+const unsigned int offset = 0;
+
+//Car creation
+ArduinoRuntime arduinoRuntime;
+BrushedMotor leftMotor{arduinoRuntime, smartcarlib::pins::v2::leftMotorPins};
+BrushedMotor rightMotor{arduinoRuntime, smartcarlib::pins::v2::rightMotorPins};
+DifferentialControl control{leftMotor, rightMotor};
+DirectionalOdometer odometer(arduinoRuntime, smartcarlib::pins::v2::leftOdometerPins, []() {odometer.update();}, PULSES_PER_METER);
+GY50 gyroscope(arduinoRuntime, offset);
+SR04 ultrasonic(arduinoRuntime, TRIGGER_PIN, ECHO_PIN, MAX_DISTANCE);
```

Comment:
The maximum distance the sensor can detect is `400`

Reply:
Noted, thanks. Seems like something which would be useful to have specified in the [API docs](https://platisd.github.io/smartcar_shield/class_s_r04.html)?

Reply:
It's already in the [SR04 instruction manuals](https://lastminuteengineers.com/arduino-sr04-ultrasonic-sensor-tutorial/) that are widely available and on [smce-gd documentation](https://github.com/ItJustWorksTM/smce-gd/wiki/Vehicle-Capabilities#ultrasonic-distance-sensor).

Reply:
Good point, thanks for the links.